meta/saiserializetest.c

```diff
@@ -1236,21 +1236,6 @@ void test_serialize_notifications()
     res = sai_serialize_switch_state_change_notification(buf, switch_id, SAI_SWITCH_OPER_STATUS_UP);
     ret = "{\"switch_id\":\"oid:0x123abc\",\"switch_oper_status\":\"SAI_SWITCH_OPER_STATUS_UP\"}";
     ASSERT_STR_EQ(buf, ret, res);
-
```

Comment:
tam_event_notify attribute and notification is still present in the switch, please bring back those tests and correct them to pass

Reply:
sai_tam_event_notification_fn() is changed to use opaque buffer (serialized as GPB binary encoded) hence the test case which uses “typedef struct _sai_tam_threshold_breach_event_t” is not applicable.

For this reason I have removed this test case.